00_Intro/bedrock_boto3_setup.ipynb

```diff
@@ -490,7 +519,7 @@
     "accept = \"application/json\"\n",
     "contentType = \"application/json\"\n",
     "\n",
-    "response = boto3_bedrock.invoke_model(\n",
+    "response = bedrock_runtime.invoke_model(\n",
```

Comment:
Line 178 markdown should also be updated to clarify that `invoke_model` is avaliable under the `Bedrock Runtime client`

Reply:
Updated the markdown